packages/pluggable-widgets-tools/package.json

```diff
@@ -46,7 +46,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "@types/react": "^18.0.0",
+    "@types/react": "18.2.0",
```

Comment:
Was the removal of caret intentional? Because this new semver is now a strict constraint. It will only and only get that version IIRC.

Reply:
No, it was a mistake. Good catch, fixed! ❇️